pyiron_atomistics/atomistics/job/interactive.py

```diff
@@ -266,27 +266,8 @@ def interactive_volume_getter(self):
         return self.initial_structure.get_volume()
 
     def _update_previous_structure(self):
-        """
-        Update the previous structure to the last step configuration
-        Args:
-            wrap_atoms (bool):
-        """
-        try:
-            indices = self.output.indices[-1]
-            positions = self.output.positions[-1]
-            cell = self.output.cells[-1]
-        except IndexError:
-            return
-        if len(self._interactive_species_lst) == 0:
-            el_lst = [el.Abbreviation for el in self.structure.species]
-        else:
-            el_lst = self._interactive_species_lst.tolist()
-        self._structure_previous = self._structure_previous.__class__(
-            positions=positions,
-            cell=cell,
-            indices=indices,
-            species=[self._periodic_table.element(el) for el in el_lst],
-        )
+        """Update the previous structure to the last step configuration."""
+        self._structure_previous = self.structure.copy()
```

Comment:
This is not the same - for example when running `calc_minimize()` in interactive mode.